optaplanner-examples/src/main/java/org/optaplanner/examples/nurserostering/domain/solver/ShiftAssignmentDifficultyComparator.java

```diff
@@ -30,10 +30,12 @@ public int compare(ShiftAssignment a, ShiftAssignment b) {
         Shift aShift = a.getShift();
         Shift bShift = b.getShift();
         return new CompareToBuilder()
-                    .append(bShift.getShiftDate(), aShift.getShiftDate()) // Descending
-                    .append(bShift.getShiftType(), aShift.getShiftType()) // Descending
                     // For construction heuristics, scheduling the shifts in sequence is better
```

Comment:
If it's indeed better, we should remove this comment.

In 2010, I ran with the date/type first way and your way, and on the datasets of this example, the benchmarker said that by date/type first way was better...